cypress/e2e/home_spec.cy.js

```diff
@@ -19,53 +19,91 @@ describe('As a user, when I load the application, I can see a collection of pain
       }
     )
     cy.visit("http://localhost:3000/")
-    cy.get(".error-text").contains(` We looked all over, but the page seems to have gotten away from us, Try
-    This one`);
+    cy.get(".error-text").contains(`We looked all over, but the page seems to have gotten away from us! Try
+    this one:`);
   });
 
+  
+  it('should bring a user to the error page if the user types in a bad URL', () => {
+    cy
+      .visit('http://localhost:3000/fsdf')
+      .get('h1').contains('Page not found')
+      .get('p.error-text').contains(`We looked all`)
+      .get('button.error-button').contains('Hom')
+    
+  })
+  
   it('should display the navBar with application logo and buttons', () => {
     cy
-      .get('.logo-img').should('be.visible')
-      .get('.navbar-link').should('be.visible')
+    .get('.logo-img').should('be.visible')
+    .get('.navbar-link').should('be.visible')
   })
-
+  
   it('should be able to click on the myGallery button and be taken to myGallery', () => {
     cy
-      .get('.button')
-      .click()
+    .get('.button')
+    .click()
   })
-
-  it('should display the aside with an h1 that read "Art Gallery" and a p tag with directions for the site', () => {
+  
+  it('should display the aside that read "Art Gallery" and a p tag with directions for the site', () => {
     cy
-      .get('h1').contains('Art Gallery')
-      .get('.description > p').contains('Art Party is how the most sophisticated, discerning art history buffs get their meme jollies. Simply click on your favorite oeuvre to get started!')
+    .get('h2').contains('Art Party')
+    .get('.description > p').contains('is how the most sophisticated, discerning art history buffs get their meme jollies. Simply click on your favorite oeuvre to get started!')
   })
-
+  
   it('should display all paintings', () => {
     cy
-      .get('.images-container').should('exist')
-      .get(':nth-child(1) > .images').should('be.visible')
-      .get(':nth-child(2) > .images').should('be.visible')
+    .get('.images-container').should('exist')
+    .get(':nth-child(1) > .images').should('be.visible')
+    .get(':nth-child(2) > .images').should('be.visible')
   })
   
-  it('should not display details for an individual painting', () => {
+  it('should be able to click on a painting a view details about it', () => {
```

Comment:
I did some poking around to see what was going on with the last test, the one you have skipped. I noticed that in the Cypress window, this test has the same /undefined url, so I think that this test would be failing too except that you don't actually have any assertions after the browser navigates to the individual page view (which never loads). I think we need to get some more help to figure out why we can't fetch an individual painting info.

Reply:
That fix worked! The test is now passing!